app/src/test/java/com/codesoom/assignment/application/UserCommandServiceTest.java

```diff
@@ -0,0 +1,250 @@
+
+    @Autowired
+    UserCommandService userCommandService;
+
+    @Nested
+    @DisplayName("createUser 메서드는")
+    class Describe_createUser {
+        @Nested
+        @DisplayName("User 가 주어진다면")
+        class Context_with_user {
+            private UserCreateRequest requestUser;
+
+            @BeforeEach
+            void setUp() {
+                requestUser = UserCreateRequest.builder()
+                        .email("a@a.com")
+                        .name("김 코")
+                        .password("123")
+                        .build();
+            }
+
+            @AfterEach
+            void after() {
+                userCommandService.deleteAll();
+            }
+
+            @Test
+            @DisplayName("User 를 저장하고 리턴한다")
+            void it_returns_user() {
+                UserResponse savedUser = userCommandService.createUser(requestUser);
+
+                assertThat(savedUser).usingRecursiveComparison()
+                        .ignoringFields("id")
+                        .isEqualTo(requestUser);
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("updateUser 메서드는")
+    class Describe_updateUser {
+        @Nested
+        @DisplayName("요청하는 User 가 존재하는 경우")
+        class Context_with_user {
+            private UserResponse savedUser;
+            private UserUpdateRequest requestUser;
+
+            @BeforeEach
+            void setUp() {
+                savedUser = userCommandService.createUser(
+                        UserCreateRequest.builder()
+                                .email("before@before.com")
+                                .name("김 코")
+                                .password("before")
+                                .build()
+                );
+
+                requestUser = UserUpdateRequest.builder()
+                        .name("김 딩")
+                        .password("after")
+                        .build();
+            }
+
+            @AfterEach
+            void after() {
+                userCommandService.deleteAll();
+            }
+
+            @Test
+            @DisplayName("User 를 수정하고 리턴한다")
+            void it_returns_updated_user() {
+                UserResponse updatedUser = userCommandService.updateUser(savedUser.getId(), requestUser);
+
+                assertThat(updatedUser).usingRecursiveComparison()
+                        .ignoringFields("id", "email")
+                        .isEqualTo(requestUser);
+            }
+        }
+
+        @Nested
+        @DisplayName("요청하는 User 가 존재하지 않는 경우")
+        class Context_with_non_existence_user {
+            private UserUpdateRequest requestUser;
+
+            @BeforeEach
+            void setUp() {
+                requestUser = UserUpdateRequest.builder()
+                        .name("김 코")
+                        .password("after")
+                        .build();
+            }
+
+            @Test
+            @DisplayName("사용자가 존재하지 않다는 예외를 던진다")
```

Comment:
클래스 이름을 다시 한 번 잘 읽어 봅시다.
```suggestion
            @DisplayName("사용자를 찾지 못했다는 예외를 던진다")
```

Reply:
5a66dd82d60722145a9dd1624a9f795fbab6088b 수정했습니다!